src/patternfly/components/Popover/popover.scss

```diff
@@ -67,38 +94,90 @@
     .pf-c-popover__arrow {
       bottom: 0;
       left: 50%;
-      transform: translateX(var(--pf-c-popover__arrow--m-top--TranslateX)) translateY(var(--pf-c-popover__arrow--m-top--TranslateY)) rotate(var(--pf-c-popover__arrow--m-top--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-top--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-top--TranslateY)) rotate(var(--pf-c-popover__arrow--m-top--Rotate));
     }
   }
 
   &.pf-m-bottom {
     .pf-c-popover__arrow {
       top: 0;
       left: 50%;
-      transform: translateX(var(--pf-c-popover__arrow--m-bottom--TranslateX)) translateY(var(--pf-c-popover__arrow--m-bottom--TranslateY)) rotate(var(--pf-c-popover__arrow--m-bottom--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-bottom--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-bottom--TranslateY))
+        rotate(var(--pf-c-popover__arrow--m-bottom--Rotate));
     }
   }
 
   &.pf-m-left {
     .pf-c-popover__arrow {
       top: 50%;
       right: 0;
-      transform: translateX(var(--pf-c-popover__arrow--m-left--TranslateX)) translateY(var(--pf-c-popover__arrow--m-left--TranslateY)) rotate(var(--pf-c-popover__arrow--m-left--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-left--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-left--TranslateY)) rotate(var(--pf-c-popover__arrow--m-left--Rotate));
     }
   }
 
   &.pf-m-right {
     .pf-c-popover__arrow {
       top: 50%;
       left: 0;
-      transform: translateX(var(--pf-c-popover__arrow--m-right--TranslateX)) translateY(var(--pf-c-popover__arrow--m-right--TranslateY)) rotate(var(--pf-c-popover__arrow--m-right--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-right--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-right--TranslateY)) rotate(var(--pf-c-popover__arrow--m-right--Rotate));
+    }
+  }
+
+  &.pf-m-success {
+    --pf-c-popover__icon--Color: var(--pf-c-popover--m-success__icon--Color);
+
+    .pf-c-title {
```

Comment:
If you use the modal component as an example, I think a good way to organize this would be to add a `.pf-m-icon` class to the `.pf-c-popover` element, which sets `display: flex;` on `.pf-c-title`. That provides an opt-in to the display change on the title to keep from breaking existing users, and it allows us to place a `<span class="pf-c-popover__title-icon">` element in the title for the icon and a `<span class="pf-c-popover__title-text">` in the title for the text.

Then instead of styling `.pf-c-title` inside of each `pf-m-[alert status]` modifier, you can set `margin-left` on `.pf-c-popover__title-text` once.

You can then also introduce a variable to set the text color (`--pf-c-popover--m-success__title-text--Color`) to control the color instead of needing to introduce variations on the title component itself, to keep the changes localized to the popover component.

Reply:
Thank you! This is fixed now. I used  modalBox component as an example and reworked the popover variations.